code/modules/food_and_drinks/food/snacks_pizza.dm

```diff
@@ -191,8 +192,12 @@
 
 /obj/item/reagent_containers/food/snacks/proc/i_kill_you(obj/item/I, mob/user)
 	if(istype(I, /obj/item/reagent_containers/food/snacks/pineappleslice))
-		to_chat(user, "<font color='red' size='7'>If you want something crazy like pineapple, I kill you.</font>")
-		user.gib() //if you want something crazy like pineapple, i kill you
+		to_chat(user, "<font color='red' size='7'>If you want something crazy like pineapple, I'll kill you.</font>") //this is in bigger text because it's hard to spam something that gibs you, and so that you're perfectly aware of the reason why you died
+		user.gib() //if you want something crazy like pineapple, i'll kill you
+	else if(istype(I, /obj/item/reagent_containers/food/snacks/grown/mushroom) && iscarbon(user))
+		to_chat(user, "<font color='red' size='4'>So, if you want mushroom, shut up.</font>") //not as large as the pineapple text, because you could in theory spam it
```

Comment:
```suggestion
		to_chat(user, "<span class='userdanger'>So, if you want mushroom, shut up.</span>") //not as large as the pineapple text, because you could in theory spam it
```